src/Pages/Account/Account.js

```diff
@@ -1,13 +1,29 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import "./Account.scss";
+import InfoBox from "./Component/infoBox";
 
 class Account extends Component {
-    render() {
-        return (
-            <div>
-                {/* //account */}
+  render() {
+    return (
+      <div className="myAccount">
+        <div className="myAccountTop">
+          <div className="myAccountTextAndBtn">
```

Comment:
- nesting 사용하고 있기 때문에 클래스명은 이렇게 길어질 필요가 없습니다!
```suggestion
        <div className="top">
          <div className="textAndBtn">
```
- top, left, first, second 등은 좋은 명명법이 아닙니다. 보다 내용에 충실한 이름으로 바꿔주세요!

Reply:
수정했습니다!